### DISCUSSION TITLE
Reduce animation does not work in dark mode

### DISCUSSION THREAD

**Discussion by agora-coffee** [lobehub/sd-webui-lobe-theme#444] [Q&A] [answered]
It seems that the 'Reduce Animation' option does not work in dark mode. You see a dimmed light of the same color as the selected 'Primary Color' on the top right, which is uncomfortable to work with. Can a fix be added for the next update?
Thank you

**Comment by agora-coffee** [comment]
The problem seems to be that the switch does not act on the following style sheet statement:
  <div class="acss-6w62k7"x/div>
      background: radial-gradient(600px at calc(100% - 300px) 300px, rgb(227, 75, 169), transparent);

**Comment by canisminor1990** [comment, marked as answer]
Reducing the animation will eliminate the flowing gradient animation in the background, while still retaining the static gradient background.
